azkaban-web-server/src/main/java/azkaban/flowtrigger/quartz/FlowTriggerScheduler.java

```diff
@@ -112,16 +119,12 @@ public void scheduleAll(final Project project, final String submitUser)
   }
 
   public void pauseFlowTrigger(final int projectId, final String flowId) throws SchedulerException {
-    logger.info(String.format("pausing flow trigger for [projectId:%s, flowId:%s]", projectId,
-        flowId));
```

Comment:
Don't we need this anymore?

Reply:
logging is moved out to function caller.